content/docs/guides/incremental-datasets.md

```diff
@@ -111,6 +111,26 @@ This dataset may not exist in the warehouse yet. That's OK, because the `WHERE`
   Dataform Web.
 </div>
 
+## A merge modification
```

Comment:
I think a concrete example of where you would use this would be useful. For example, if your rows for transactions get updated historically. Walk through an example where you have a config like:

```
config {
  type: "incremental",
  uniqueKey: "transaction_id",
}

select * from raw.transactions
${when(incremental(), "where last_modified > select max(last_modified) from ${self()}")}
```

Reply:
I added a new section with an example, does read nicer